### PR TITLE
[Tizen Studio] Attach-Debugging Notes section update

### PR DESCRIPTION
**Pull request by manishtr**
Signed-off-by: manish.r <manish.r@samsung.com>

### Change Description ###

Updated the Notes section for Native Attach-Debugging in Tizen Studio

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

File: docs/application/native/tutorials/process/debugging-app.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ To debug an application that is already running on the emulator or a device:
 
 - In the **Project Explorer** view, right-click the project and select **Debug As > Tizen Native Application - Attach**.
 
-  Note that the source code of the project must be related to the application that is running on the target.
+  Note that the source code of the project must be related to the application that is running on the target also as soon as the attach happens then the code will be stopped on some address which will belong to the internal code of main loop hence its code will not be visible, you can add a breakpoint in the application code which can be hit later on and click on continue button to resume debugging.
```
Comment (dkson95) [inline docs/application/native/tutorials/process/debugging-app.md:53]:
```suggestion
  Note that the source code of the project must be related to the application that is running on the target. Also, once the attach happens, the debugger can stop at arbitrary address which is belong to the internal code of main loop, hence this code code might not be visible.  In this case, you can add a breakpoint in the application code which can be hit later on and click `continue` button to resume debugging.
```

File: docs/application/native/tutorials/process/debugging-app.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ To debug an application that is already running on the emulator or a device:
 
 - In the **Project Explorer** view, right-click the project and select **Debug As > Tizen Native Application - Attach**.
 
-  Note that the source code of the project must be related to the application that is running on the target also as soon as the attach happens then the code will be stopped on some address which will belong to the internal code of main loop hence its code will not be visible, you can add a breakpoint in the application code which can be hit later on and click on continue button to resume debugging.
+  Note that the source code of the project must be related to the application that is running on the target. Also, once the attach happens, the debugger can stop at arbitrary address which is belong to the internal code of main loop, hence this code code might not be visible.  In this case, you can add a breakpoint in the application code which can be hit later on and click `continue` button to resume debugging.
```
Comment (dkson95) [inline docs/application/native/tutorials/process/debugging-app.md:53]: oh noticed `code` twice !
s/code code/code/g

Thank you.

**Review by dkson95 — CHANGES_REQUESTED** [review]

File: docs/application/native/tutorials/process/debugging-app.md (hunk 2 above)
Comment (manishtr) [reply docs/application/native/tutorials/process/debugging-app.md]: updated Mr Son, Thanks for the review

**Review by dkson95 — APPROVED** [review]
thank you!

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@manishtr Request you to kindly see the suggestion.

**Review by nawab-samsung — APPROVED** [review]
@manishtr Thanks for incorporating the comment.

**Pull request merged**